app/src/main/res/xml/settings_preference_fragment.xml

```diff
@@ -1,18 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
-    <PreferenceCategory android:title="App Settings">
+    <PreferenceCategory android:title="General">
```

Comment:
@Avjeet Don't you think "App Settings" is more suitable as a title?

Reply:
@harsh-2711 almost all the apps have the Settings as there title and moreover if I change the title to app setting I also have to change the name in the navigation panel and in overflow menu in lux meter activity and also it doesn't make much of the difference